biz.aQute.repository/src/aQute/bnd/repository/maven/pom/provider/BndPomRepository.java

```diff
@@ -51,9 +54,9 @@
 	private String				name;
 	private Reporter			reporter			= new Slf4jReporter(BndPomRepository.class);
 	private InnerRepository	repoImpl;
-	private Revision			revision;
+	private Set<Revision>		revisions;
```

Comment:
Does this need to be a Set? Are we worried about dups? We need a well defined iteration order. Use List.